### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by jamtur01**
Minor edits to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,18 +52,20 @@ This will collect one bean and override its service name.
 
 ## Usage
 
-### `(mbeans/instrumentation opts)`
-### `(mbeans/instrumentation)`
```
Comment (faxm0dem) [inline README.md:56]: this function accepts one or zero arguments

**Comment by faxm0dem** [comment]
other than my inline comment, LGTM thanks!
can you just change that and I'll merge

**Comment by jamtur01** [comment]
Updated.

**Comment by faxm0dem** [comment]
Thanks!

**Pull request merged**